tests/converters/table-post-fixer.js

```diff
@@ -0,0 +1,426 @@
+				[ '21', '22' ]
+			] ), model.schema );
+
+			model.change( writer => {
+				writer.remove( Range.createIn( root ) );
+				writer.insert( parsed, root );
+			} );
+
+			expect( formatTable( getModelData( model, { withoutSelection: true } ) ) ).to.equal( formattedModelTable( [
+				[ { colspan: 6, contents: '00' } ],
+				[ { rowspan: 2, contents: '10' }, '11', { colspan: 3, contents: '12' }, '' ],
+				[ '21', '22', '', '', '' ]
+			] ) );
+		} );
+
+		it( 'should fix wrong rowspan attribute on table header', () => {
+			const parsed = parse( modelTable( [
+				[ { rowspan: 2, contents: '00' }, { rowspan: 3, contents: '01' }, '02' ],
+				[ { rowspan: 8, contents: '12' } ],
+				[ '20', '21', '22' ]
+			], { headingRows: 2 } ), model.schema );
+
+			model.change( writer => {
+				writer.remove( Range.createIn( root ) );
+				writer.insert( parsed, root );
+			} );
+
+			expect( formatTable( getModelData( model, { withoutSelection: true } ) ) ).to.equal( formattedModelTable( [
+				[ { rowspan: 2, contents: '00' }, { rowspan: 2, contents: '01' }, '02' ],
+				[ '12' ],
+				[ '20', '21', '22' ]
+			], { headingRows: 2 } ) );
+		} );
+
+		it( 'should fix wrong rowspan attribute on table body', () => {
+			const parsed = parse( modelTable( [
+				[ '00', '01', '02' ],
+				[ { rowspan: 2, contents: '10' }, { rowspan: 3, contents: '11' }, '12' ],
+				[ { rowspan: 8, contents: '22' } ]
+			], { headingRows: 1 } ), model.schema );
+
+			model.change( writer => {
+				writer.remove( Range.createIn( root ) );
+				writer.insert( parsed, root );
+			} );
+
+			expect( formatTable( getModelData( model, { withoutSelection: true } ) ) ).to.equal( formattedModelTable( [
+				[ '00', '01', '02' ],
+				[ { rowspan: 2, contents: '10' }, { rowspan: 2, contents: '11' }, '12' ],
+				[ '22' ]
+			], { headingRows: 1 } ) );
+		} );
+
+		it( 'should fix multiple tables', () => {
+			const tableA = modelTable( [
+				[ '11' ],
+				[ '21', '22' ]
+			] );
+			const tableB = modelTable( [
+				[ 'aa' ],
+				[ 'ba', 'bb' ]
+			] );
+
+			const parsed = parse( tableA + tableB, model.schema );
+
+			model.change( writer => {
+				writer.remove( Range.createIn( root ) );
+				writer.insert( parsed, root );
+			} );
+
+			const expectedTableA = formattedModelTable( [
+				[ '11', '' ],
+				[ '21', '22' ]
+			] );
+			const expectedTableB = formattedModelTable( [
+				[ 'aa', '' ],
+				[ 'ba', 'bb' ]
+			] );
+
+			expect( formatTable( getModelData( model, { withoutSelection: true } ) ) ).to.equal( expectedTableA + expectedTableB );
+		} );
+
+		it( 'should not crash on table remove', () => {
+			setModelData( model, modelTable( [
+				[ '11', '12' ]
+			] ) );
+
+			expect( () => {
+				model.change( writer => {
+					writer.remove( Range.createIn( root ) );
+				} );
+			} ).to.not.throw();
+
+			expect( getModelData( model, { withoutSelection: true } ) ).to.equal( '<paragraph></paragraph>' );
+		} );
+	} );
+
+	describe( 'collaboration', () => {
```

Comment:
The `it()` descriptions are not very helpful in this section. They should explain what the code is supposed to do, not simple ids of tests.